huaweicloud/services/acceptance/apig/resource_huaweicloud_apig_response_test.go

```diff
@@ -197,15 +193,15 @@ resource "huaweicloud_apig_response" "test" {
     status_code = 401
   }
 }
-`, testAccApigResponse_base(rName), rName)
+`, testAccResponse_base(name), name)
 }
 
-func testAccApigResponse_update(rName string) string {
+func testAccResponse_update(name string) string {
```

Comment:
the names of base resources should not be updated

Reply:
There is no test function to reference this function.

Reply:
done.